### DISCUSSION TITLE
Add Pit Lane to Track Map!

### DISCUSSION THREAD

**Discussion by TheBaconCat** [f1multiviewer/issue-tracker#223] [Feature requests] [answered]
Currently when viewing the Track Map there is no pit lane. If it is possible just like the start/finish line it would be nice to add the pit lanes to all tracks to have a visual visual for cars coming and going during practice & qualifying and obviously pitting during the race!
![5AE88E10-69D0-4BE4-B4F1-333ACEE56A74](https://user-images.githubusercontent.com/11319293/192955076-50fa44a7-0412-443e-9a5b-81e54771a62f.png)

**Comment by jeppeolesen** [comment, marked as answer]
This has been implemented and released in MultiViewer 2.0, which is now available in the beta release channel.

In order to use the pit lane feature, you need to be a sponsor.

![image](https://github.com/user-attachments/assets/378e798c-fcd2-4300-ae7b-ee15a53caaad)
